article/app/views/fragments/liveBlogNavigation.scala.html

```diff
@@ -45,7 +45,7 @@
                 href="/@{article.content.metadata.id}@oldest.suffix"
                 title="Go to page @oldest.pageNumber"
             }
-            class="liveblog-navigation__link link-navigation__link--secondary liveblog-navigation__link--secondary--older @if(pagination.oldest.isEmpty) { liveblog-navigation__link--disabled }"
+            class="liveblog-navigation__link link-navigation__link--secondary link-navigation__link--secondary--older @if(pagination.oldest.isEmpty) { liveblog-navigation__link--disabled }"
```

Comment:
This should be `liveblog-navigation__link--secondary--older`, I just forgot to rename `link-navigation__link--secondary` to `liveblog-navigation__link--secondary`…